### PR TITLE
VW MQB: VW Jetta Mk7, Škoda Kodiaq Mk1

### PR DESCRIPTION
**Pull request by jyoung8607**
**Volkswagen Jetta Mk7**

Covers all variants of the seventh generation Jetta (2018-present), including the Jetta GLI and the Volkswagen Sagitar in China.

- [x] added to README
- [x] test route added to [test_car_models](../../selfdrive/test/test_car_models.py)
- [x] route with openpilot: 58a7d3b707987d65|2021-03-25--17-26-37

Thanks to community Jetta owners oxymoron (2018), v3lasco (2019), salryujo (2020), and Antilag (2021).

**Škoda Kodiaq Mk1**

Covers all variants of the first generation Kodiaq (2016-present).

- [x] added to README
- [x] test route added to [test_car_models](../../selfdrive/test/test_car_models.py)
- [x] route with openpilot: 90434ff5d7c8d603|2021-03-15--12-07-31

Thanks to community Kodiaq owners Gold (2018) and Saber422 (another 2018).

### CODE REVIEW

File: selfdrive/test/test_car_models.py [hunk 1]
```diff
@@ -441,6 +441,14 @@ def get_route_log(route_name):
     'carFingerprint': VOLKSWAGEN.GOLF,
     'enableCamera': True,
   },
+  "58a7d3b707987d65|2021-03-25--17-26-37": {
+    'carFingerprint': VOLKSWAGEN.JETTA_MK7,
+    'enableCamera': True,
+  },
+  "ea70677b54a1bca1|2020-06-02--16-55-55": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_car_models.py:448]: this one's gone

Comment (jyoung8607) [reply selfdrive/test/test_car_models.py]: Pulled Passat till I can get a route from one of the two owners tomorrow AM, I'll put it in a separate PR then.

**Comment by adeebshihadeh** [comment]
Test failed because of duplicate FW versions.

File: README.md [hunk 2]
```diff
@@ -169,10 +169,12 @@ Community Maintained Cars and Features
 | Nissan    | Leaf 2018-20                  | ProPILOT          | Stock            | 0mph               | 0mph         |
 | Nissan    | Rogue 2018-19                 | ProPILOT          | Stock            | 0mph               | 0mph         |
 | Nissan    | X-Trail 2017                  | ProPILOT          | Stock            | 0mph               | 0mph         |
+| Škoda     | Kodiaq 2018                   | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Subaru    | Ascent 2019                   | EyeSight          | Stock            | 0mph               | 0mph         |
 | Subaru    | Crosstrek 2018-19             | EyeSight          | Stock            | 0mph               | 0mph         |
 | Subaru    | Forester 2019-20              | EyeSight          | Stock            | 0mph               | 0mph         |
 | Subaru    | Impreza 2017-19               | EyeSight          | Stock            | 0mph               | 0mph         |
+| Volkswagen| Jetta 2018-21                 | Driver Assistance | Stock            | 0mph               | 0mph         |
```
Comment (adeebshihadeh) [inline README.md:177]: has it been tested on all these years?

Comment (jyoung8607) [reply README.md:177]: Yep. Legitimately had four testers that happened to be all four years.

**Pull request merged**